### ISSUE TITLE
Adding Custom 404 Page

### ISSUE THREAD

**Issue by jaskaranl** [akshitagupta15june/Moksh#233]
I will design an Error Page for calls or redirecting to pages which do not exist.

Hello!
I am Jaskaran Kamboj , a proud contributor at GSSoC '23. I would like to the contribute to this project by designing an Error Page for calls or redirecting to pages which do not exist.

Things I will be doing as of now:
 Designing the error page 404

**Comment by ManashAnand** [comment]
Hi there! I would like to contribute to this issue. I have gone through the problem description and would love to help out. I have experience with similar problems and believe I can provide a solution. Could you please assign this issue to me so that I can start working on it? Thank you!

**Comment by ankitjt** [comment]
@jaskaranl can you share some mock-ups for us to review ? 

@ManashAnand do you also want to just  design a mock-up or a mock-up and make the page too ?

**Comment by ManashAnand** [comment]
I will make the page too

**Comment by jaskaranl** [comment]
![Untitled 3](https://github.com/akshitagupta15june/Moksh/assets/95584564/01902d83-2dc8-4b7d-84fd-54c02af1eca8)
<img width="869" alt="Untitled" src="https://github.com/akshitagupta15june/Moksh/assets/95584564/3933fec2-df7d-4b0c-82c0-2c3bcefe0936">
![Untitled 2](https://github.com/akshitagupta15june/Moksh/assets/95584564/b608ce42-91d5-4961-b87d-3be1268a6365)

**Comment by jaskaranl** [comment]
<img width="869" alt="Untitled" src="https://github.com/akshitagupta15june/Moksh/assets/95584564/8b58043d-3533-4aa2-897c-264cfadd68f6">
<img width="1440" alt="Untitled" src="https://github.com/akshitagupta15june/Moksh/assets/95584564/1b86ee82-f8c5-471f-a81b-346627edf956">
![Untitled 3](https://github.com/akshitagupta15june/Moksh/assets/95584564/2b07c9b2-8bed-4f88-a131-ae982a39eb12)


These are some mockups let me know about it,
so that i can make the pages

**Comment by ManashAnand** [comment]
Yes, I will make the third one for you if you give me the oppurtunity.

**Comment by karthiknadar1204** [comment]
@jaskaranl The mockups look really good, please feel free to post these mock-ups in the moksh-design-moqups channel for others to get a look at it.

**Comment by karthiknadar1204** [comment]
We can then decide if any changes are required or on how do we go about with implementing it

**Comment by akshitagupta15june** [comment]
@jaskaranl please join discord channel for discussion

**Comment by jaskaranl** [comment]
> 

posted the moockup on discord :)

**Comment by ankitjt** [comment]
Hey, adding 404 page will not work as the project is hosted using GitHub pages.
